mods/cnc/maps/scb30ea/scb30ea-AI.lua

```diff
@@ -0,0 +1,421 @@
+--[[
+   Copyright (c) The OpenRA Developers and Contributors
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+GDInuke1 = { type = "nuk2", pos = CPos.New(6,3), blocked = false }
+GDIproc1 = { type = "proc", pos = CPos.New(8,12), blocked = false }
+GDIpyle = { type = "pyle", pos = CPos.New(17,4), blocked = false }
+GDIweap = { type = "weap", pos = CPos.New(10,9), blocked = false }
+GDIproc2 = { type = "proc", pos = CPos.New(12,3), blocked = false }
+GDInuke2 = { type = "nuk2", pos = CPos.New(4,11), blocked = false }
+GDInuke3 = { type = "nuk2", pos = CPos.New(3,14), blocked = false }
+GDIdef1 = { type = "atwr", pos = CPos.New(8,5), blocked = false }
+GDIdef2 = { type = "atwr", pos = CPos.New(6,10), blocked = false }
+GDIdef3 = { type = "atwr", pos = CPos.New(6,16), blocked = false }
+GDIdef4 = { type = "gtwr", pos = CPos.New(9,8), blocked = false }
+GDInuke4 = { type = "nuk2", pos = CPos.New(13,12), blocked = false }
+GDIdef5 = { type = "atwr", pos = CPos.New(15,10), blocked = false }
+GDIdef6 = { type = "atwr", pos = CPos.New(13,16), blocked = false }
+GDInuke5 = { type = "nuk2", pos = CPos.New(11,14), blocked = false }
+GDIsilo1 = { type = "silo", pos = CPos.New(3,6), blocked = false }
+GDIsilo2 = { type = "silo", pos = CPos.New(5,17), blocked = false }
+GDIdef7 = { type = "atwr", pos = CPos.New(19,5), blocked = false }
+GDIdef8 = { type = "atwr", pos = CPos.New(10,21), blocked = false }
+GDIsilo3 = { type = "silo", pos = CPos.New(17,2), blocked = false }
+GDIsilo4 = { type = "silo", pos = CPos.New(9,17), blocked = false }
+GDIdef9 = { type = "atwr", pos = CPos.New(18,20), blocked = false }
+GDIhpad1 = { type = "hpad", pos = CPos.New(3,18), blocked = false }
+
+GDIBase = { GDINuke1, GDIPyle, GDIWeap, GDINuke2, GDINuke3, GDIDef1, GDIDef2, GDIDef3, GDIDef4,
+	GDINuke4, GDIDef5, GDIDef6, GDINuke5, GDISilo1, GDISilo2, GDIDef7, GDIDef8, GDISilo3, GDISilo4, GDIDef9 }
+GDIRebuildList = { GDInuke1, GDIproc1, GDIpyle, GDIweap, GDIproc2, GDInuke2, GDInuke3, GDIdef1, GDIdef2, GDIdef3, GDIdef4,
+	GDInuke4, GDIdef5, GDIdef6, GDInuke5, GDIsilo1, GDIsilo2, GDIdef7, GDIdef8, GDIsilo3, GDIsilo4, GDIdef9 }
+BSizes = { nuk2 = CVec.New(2,3), proc = CVec.New(3,4), pyle = CVec.New(2,3), weap = CVec.New(3,3), atwr = CVec.New(1,1), gtwr = CVec.New(1,1), silo = CVec.New(2,1), hpad = CVec.New(2,2) }
+
+ProductionBuildings = { i = GDIPyle, v = GDIWeap, a = GDICYard }
+
+ProdQueue = { i = { }, v = { }, a = { } }
```

Comment:
```suggestion
ProductionQueue = { infantry = { }, vehicle = { }, aircraft = { } }
```

maybe? I had a hard time reading this.